addons/medical_status/functions/fnc_isInStableCondition.sqf

```diff
@@ -17,7 +17,28 @@
 
 params ["_unit"];
 
-alive _unit
-&& {!IS_UNCONSCIOUS(_unit)}
-&& {GET_WOUND_BLEEDING(_unit) == 0}
-&& {_unit call FUNC(hasStableVitals)}
+if (!alive _unit
+    || {IS_UNCONSCIOUS(_unit)}
+    || {IS_BLEEDING(_unit)}
+    || {GET_HEART_RATE(_unit) < EGVAR(medical_treatment,minHeartRatePAK)}
+) exitWith {false};
+
+private _requiredBloodVolume = DEFAULT_BLOOD_VOLUME * (EGVAR(medical_treatment,minBloodPAK) / 100);
+if (GET_BLOOD_VOLUME(_unit) < _requiredBloodVolume) exitWith {false};
+
+if (EGVAR(medical_treatment,allowPainPAK) == 0 && { IS_IN_PAIN(_unit) }) exitWith {false};
+if (EGVAR(medical_treatment,allowFracturesPAK) == 0 && { !(GET_FRACTURES(_unit) isEqualTo []) }) exitWith {false};
+
+// Require at least bandaged wounds.
+if (EGVAR(medical_treatment,requiredWoundStatePAK) == 1) then {
+    _openWounds = GET_OPEN_WOUNDS(_unit);
+    {
+        if(_x select 2 > 0) exitWith {false};
+    } forEach _openWounds;
+};
+
+
+// Require stitched wounds.
+if (EGVAR(medical_treatment,requiredWoundStatePAK) == 2 && { !(count GET_OPEN_WOUNDS(_unit) ==  count GET_STITCHED_WOUNDS(_unit)) }) exitWith {false};
```

Comment:
For similar reasons as the "require at least bandaged wounds" check, I think this can also fail. I think you just want to check that there are no bandaged wounds with amount > 0. [reference](https://github.com/acemod/ACE3/blob/master/addons/medical_treatment/functions/fnc_getStitchableWounds.sqf#L5)